### PR TITLE
viewer/editor Initial fix for warning when viewing big jpg #1217, vie…

### PR DESCRIPTION
**Pull request by pskowronek**
Initial fix for warning when viewing big images #1217 plus viewer/editor unification/refactor

### CODE REVIEW

**Comment by hajdam** [comment]
Hi, I think there were some discussion about this few years back. I don't mind this fix, just few notes:

- Maybe better to avoid "File Open" terminology as there is different concept for opening files in muCommander and stick with "File Viewing"
- My vote still goes to remove showing pop-up dialog altogether and instead just show warning / continue button in viewer itself

**Comment by pskowronek** [comment]
> * Maybe better to avoid "File Open" terminology as there is different concept for opening files in muCommander and stick with "File Viewing"

'FileOpen' is for me intuitive (especially that it is under `viewer` packager. I'm open to rename it, since I don't have better name :) if anyone is able to suggest sth nicer then please.

> * My vote still goes to remove showing pop-up dialog altogether and instead just show warning / continue button in viewer itself

Yeah, in another PR I could change that, plus move the msg box somewhere else in the code.

**Review by hajdam — APPROVED** [review]

File: package/readme.txt [hunk 1]
```diff
@@ -41,7 +41,7 @@ Localization:
 -
 
 Bug fixes:
--
+- Don't show warning message about too large file while previewing images files (ticket #1217)
```
[the remark this answers is not in the document]
Comment (pskowronek) [reply package/readme.txt]: ok, PR updated

**Review by ahadas — APPROVED** [review]

File: package/readme.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahadas) [reply package/readme.txt]: thanks!

**Pull request merged**